extensions/amp-bind/0.1/test/integration/test-bind-impl.js

```diff
@@ -619,7 +741,10 @@ describe
             env,
             container,
             '[checked]="true" [disabled]="false" disabled',
-            /* opt_tagName */ 'input'
+            {
+              tag: 'input',
+              insertQuerySelectorAttr: useQuerySelector,
```

Comment:
From purely a practical standpoint, do you think it would be cleaner to make a module level `optimized` var consumed by the `createElement` helper to conditionally add the css class? It breaks DI, but cuts down on boilerplate in each test.